tests/python_client/milvus_client/test_milvus_client_timestamptz.py

```
@@ -1308,4 +1099,297 @@ def test_milvus_client_timestamptz_add_field_not_nullable(self):
+    #########################################################
+    """
+    @pytest.fixture(scope="module", autouse=True)
+    def prepare_timestamptz_collection(self, request):
+        """
+        Prepare timestamptz collection for the tests
+        """
+        default_dim = 3
+        client = self._client()
+        collection_name = COLLECTION_NAME
+        schema = self.create_schema(client, enable_dynamic_field=False)[0]
+        schema.add_field(default_primary_key_field_name, DataType.INT64, is_primary=True, auto_id=False)
```

Ruff (0.14.10)

`DataType` may be undefined, or defined from star imports

(F405)